doc/source/install/debian.rst

```diff
@@ -4,6 +4,25 @@ Debian GNU/Linux
 This section describes how to install Mroonga related deb packages on
 Debian GNU/Linux. You can install them by ``apt``.
 
+bookworm (MariaDB)
+------------------
+
+Install::
+
+  % sudo apt update
+  % sudo apt install -y -V apt-transport-https
+  % sudo apt install -y -V wget
```

Comment:
We also enable Apache Arrow repository: https://groonga.org/docs/install/debian.html#bookworm

Reply:
Thank you for your comment.
I've fix it.